tests/pages/nodes/NodesPage-cy.js

```diff
@@ -1,97 +1,117 @@
 describe('Nodes Page', function () {
 
-  beforeEach(function () {
-    cy.configureCluster({
-      mesos: '1-for-each-health',
-      nodeHealth: true
-    });
-    cy.visitUrl({url: '/nodes'});
-  });
-
```

Comment:
Moved this code under the `Filters nodes table` context because in the `Nodes grid` context we are doing a different call to `visitUrl` we should do a best-effort to only call `visitUrl` once per test as this is expensive and time consuming and can easily cause each test to take an extra couple seconds to run.